templates/blocks/share_buttons.twig

```diff
@@ -1,27 +1,27 @@
 <div class="share-buttons">
 	<!-- Whatsapp -->
-	<a href="https://wa.me/?text={{ social.link }}"
+	<a href="https://wa.me/?text={{ social.link }}&utm_source=whatsapp&utm_medium={{ utm_medium }}{{ utm_campaign_param }}"
```

Comment:
Shouldn't `utm_content` also be part of these urls?

Reply:
As far as I understand, we don't need to include it if the local campaign isn't set.

> The utm_campaign value should be based on the “Local Project” value defined in Planet 4 Analytics settings. All values should be in lowercase and spaces should be transformed to dashes . If no value is selected, we don’t need to include the utm_campaign value. 

Correct me if I'm wrong

Reply:
Good catch! I missed to add it.

Reply:
@comzeradd added